opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -995,8 +995,17 @@ def record_exception(
                 type(exception), value=exception, tb=exception.__traceback__
             )
         )
+        module = (
+            exception.__module__
+            if exception.__class__.__module__ != "builtins"
+            else ""
+        )
+        exc_type = (
+            f"{module}.{exception.__class__.__qualname__}"
```

Comment:
What about using `type(exception).__qualname__` that is a bit shorter?

Reply:
Agreed